tests/kernel/install_ltp.pm

```diff
@@ -275,6 +275,13 @@ sub run {
 
     add_we_repo_if_available;
 
+    my $grub_param;
+    my $grub_param = "console=hvc0"     if (get_var('ARCH') eq 'ppc64le');
+    my $grub_param = "console=ttysclp0" if (get_var('ARCH') eq 's390x');
```

Comment:
@rwx788 Thanks for review! I've fixed duplicate `my` (copy paste error). `bootmenu_type_console_params()` is not suitable much for us: `install_ltp.pm` installs LTP and publish it (new qcow image) and then it's run by more than 100 tests. So we don't want to use slow `bootmenu_type_console_params()`. And we already change grub setup for IMA tests (`ltp_ima`) + there are going to be more changes (missing timestamp on dmesg on some archs, which needs to be detected after system is booted - this might be the most important thing why change grub config).

Reply:
Sounds good! We already have rewritten something like bootmenu_type_console_params to just return string in the end, so you could reuse it here, but that's just an idea. Always typing doesn't scale.

Reply:
@rwx788 @richiejp I've added one more commit 2b8241bca ("LTP: Add timestamp to dmesg on serial terminal"), which adds more config. I'll post verify runs in few minutes.